app/scrapers/yahoo.py

```diff
@@ -24,12 +24,12 @@ def parseResponse(self, soup):
                 f = r.split('RU=')
                 e = f[-1].split('/RK=0')
                 u = e[0].replace('%3a', ':').replace('%2f', '/') \
-                        .replace('%28', '(').replace('%29', ')') \
-                        .replace('%3f', '%3d', '=').replace('%26', '&') \
-                        .replace('%29', ')').replace('%26', "'") \
-                        .replace('%21', '!').replace('%23', '$') \
-                        .replace('%40', '[').replace('%5b', ']')
-
+                    .replace('%28', '(').replace('%29', ')') \
+                    .replace('%3f', '?').replace('%3d', '=') \
+                    .replace('%26', '&').replace('%29', ')') \
+                    .replace('%26', "'").replace('%21', '!') \
```

Comment:
`%26` only translate to `&`, I don't know why we are replacing it with `'`

Reply:
It was already there, even I also thought of that. Do I remove it ?

Reply:
This code is actually just parsing URL, so we can just use already defined methods instead of maually figuring out things https://docs.python.org/3/library/urllib.parse.html#urllib.parse.unquote.

Reply:
This works nicely
`urllib2.unquote(e[0])`
Do i change like this ?

Reply:
urllib.parse.unquote seems fine to me

Reply:
urllib has no module parse, but `urllib.unquote` works fine